### ISSUE TITLE
Binary SPI communication errror

### ISSUE THREAD

**Issue by gvinet** [hydrabus/hydrafw#81]
The binary SPI communication works  for commit a2aab9dcda85359a4c4e44018ab9112924ec129c (HydraFW v0.8 Beta), but not for the latest version, 677dd8dcef7225f5c9e97ab15042f18d428b088f.

It prevents pynfcreader to work with the latest version.

* Output of bbio_hydranfc_init.py script (commit a2aab9dcda85359a4c4e44018ab9112924ec129c):
```
Configure NFC/TRF7970A in SPI mode with Chip Select
exit
Invalid command.
>   
>  gpio pa3 mode out off
Clearing PA3 
> gpio pa2 mode out on
Setting PA2 
> gpio pc0 mode out on
Setting PC0 
> gpio pc1 mode out on
Setting PC1 
> gpio pb11 mode out off
Clearing PB11 
> gpio pb11 mode out on
Setting PB11 
> gpio pa2-3 pc0-1 pb11 r
PA2	1
PA3	0
PB11	1
PC0	1
PC1	1
>   
Into BBIO mode: OK  
Switching to SPI mode: SPI1  
Configure SPI2 polarity 0 phase 1: OK 
Configure SPI2 speed to 2620000 bits/sec: OK 
Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 91 OK 
```

* Output of bbio_hydranfc_init.py script (commit 677dd8dcef7225f5c9e97ab15042f18d428b088f):

```
Configure NFC/TRF7970A in SPI mode with Chip Select
exit
Invalid command.
>   
>  gpio pa3 mode out off
Clearing PA3 
> gpio pa2 mode out on
Setting PA2 
> gpio pc0 mode out on
Setting PC0 
> gpio pc1 mode out on
Setting PC1 
> gpio pb11 mode out off
Clearing PB11 
> gpio pb11 mode out on
Setting PB11 
> gpio pa2-3 pc0-1 pb11 r
PA2	1
PA3	0
PB11	1
PC0	1
PC1	1
>   
Into BBIO mode: OK  
Switching to SPI mode: SPI1  
Configure SPI2 polarity 0 phase 1: OK 
Configure SPI2 speed to 2620000 bits/sec: OK 
Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error


Write TRF7970A Software Initialization 0x83 0x83 (no read): OK 
Write TRF7970A Idle 0x80 0x80 (no read): OK 
Read TRF7970A Modulator/SYS_CLK Control Register (0x09): 00 Error
```

**Comment by Baldanos** [comment]
The script has been fixed in [7a352ac74482d1a70460ab1b1d528fea30252945](https://github.com/hydrabus/hydrafw/commit/7a352ac74482d1a70460ab1b1d528fea30252945).

Some BBIO changes between 0.8 and master made this script not working (see in the [wiki](https://github.com/hydrabus/hydrafw/wiki/HydraFW-Binary-SPI-mode-guide#configure-spi-port-0b10000xyz)). It has been fixed now.

Thanks for reporting !

**Comment by gvinet** [comment]
Thank you!
